webapp/src/main/java/com/box/l10n/mojito/service/translationkit/TranslationKitService.java

```diff
@@ -292,7 +292,7 @@ public void checkForPartiallyImported(Long dropId) {
         List<TranslationKit> translationKits = translationKitRepository.findByDropId(drop.getId());
         boolean partiallyImported = false;
         for (TranslationKit translationKit : translationKits) {
-            if (!translationKit.getImported()) {
+            if (translationKit.getNumTranslationKitUnits() > 0 && !translationKit.getImported()) {
```

Comment:
@ehoogerbeets This fixes a bug I found for the partially imported drop.

Reply:
👍